lib/tower_cli/resources/group.py

```diff
@@ -52,23 +52,36 @@ class Resource(models.Resource):
                   'from the external source.')
     @click.option('--update-on-launch', type=bool, help='Refresh inventory '
                   'data from its source each time a job is run.')
-    def create(self, credential=None, source=None, **kwargs):
+    def create(self, fail_on_found=False, force_on_exists=False, **kwargs):
         """Create a group and, if necessary, modify the inventory source within
         the group.
         """
+        # Break out the options for the group vs its inventory_source
+        group_fields = [f.name for f in self.fields]
+        is_kwargs = {}
+        for field in kwargs.copy():
+            if field not in group_fields:
+                is_kwargs[field] = kwargs.pop(field)
+
+        # Handle alias for "manual" source
+        if 'source' in is_kwargs and is_kwargs['source'] == 'manual':
+            is_kwargs.pop('source')
```

Comment:
or you could do `is_kwargs.pop('source', None') if is_kwargs.get('source', None) == 'manual' else None` <-- but that is more confusing